src/org/labkey/test/components/list/ManageListsGrid.java

```diff
@@ -52,8 +53,9 @@ public File exportSelectedLists()
 
     public BeginPage deleteSelectedLists()
     {
-        deleteSelectedRows();
-        return new BeginPage(getDriver());
+        clickHeaderButtonAndWait("Delete");
```

Comment:
perhaps `clicHeaderButtonAndWait `could return a new `ConfirmDeletePage` and this method could chain those calls